docs/openhabian-DEBUG.md

```diff
@@ -0,0 +1,93 @@
+Please proceed as told there. openHABian has its own repository at https://github.com/openhab/openhabian/
+Search the issues listed there first if 'your' problem has already been seen and eventually opened as an issue by someone else. If so, you may leave a "me too" comment there but please do not open another issue to avoid duplicates.
+You can reference other issues (eventually also request to reopen closed ones) and Pull Requests by their number (just type #nnn along with your text, GitHub will insert the proper link).
+If you open an issue, we kindly ask you to deliver as much information as possible. It is awkwardly annoying if we need to spend time asking and asking what the real problem is about. Please avoid that situation, be proactive and tell us in the very first place.
+Once you opened the issue, copy `/boot/first-boot.log` from your openHABian box over to your desktop and upload it to GitHub.
+If you succeed logging on and get to see a banner with system information, please also copy that as part of your issue.
+
+If you're able to help in producing a fix to problems, we happily take any Pull Request.
+Explaining git and Github unfortunately is out of scope.
+For simple fixes to a single file only, you can click through the source starting at https://github.com/openhab/openhabian and edit the file online, GitHub will then offer to create the PR.
+You can also clone the openhabian repository, make your changes locally and use git to check in your changes and upload them to a repo copy of yours, then follow the git-offered link to create the PR. 
+Either way, don't forget to sign your work.
+
+## Checkpoint
+::: tip
+Remember to always let `openhabian-config` update itself on start.
+:::
+
+::: tip
+If you want to change anything to work around some not yet fixed bug, you can directly edit the files in and below `/opt/openhabian` on your box. Just do not let openhabian-config update itself on start as that would overwrite your changes.
+:::
+
+The main program is in `openhabian-setup.sh`. If the initial unattended install fails again and again at the same step (say Java installation), you may comment that step out. But mind the code in build-image/first-boot.bash quite towards the end starting with`git clone`. This is where openHABian updates itself. If you don't comment that out as well, it'll overwrite your changes on next install run.
+
+::: warning Disclaimer
+For obvious reasons, this is not a supported procedure. We just want to give you a hint what you _could_ try doing if your install fails
+and you're sitting there, desperately looking for a fix.
+G*oo*gle and learn yourself what you need to edit a file, learn to understand shell programming basics, you're on your own here.
+If you change openHABian code on your box, temember for the time it takes to get openHABian officially fixed, you must not let openhabian-config update itself on start as that would overwrite your changes.
```

Comment:
temember -> remember

btw: are you sure that :::warning and :::tip works correctly? if I open as file, thhis is not processed.

Reply:
no so I removed it